### ISSUE TITLE
sf project generate manifest --name package-all-org-items.xml --output-dir ./manifest --include-packages managed,unlocked --from-org - Could not infer a metadata type

### ISSUE THREAD

**Issue by Jbanniere** [forcedotcom/cli#2858]
### Summary
Error when calling sf project generate manifest on some orgs

> **IMPORTANT**
sf project generate manifest --name package-all-org-items.xml --output-dir ./manifest --include-packages managed,unlocked --from-org

### System Information
bash for windows
{
  "architecture": "win32-x64",
  "cliVersion": "@salesforce/cli/2.39.6",
  "nodeVersion": "node-v18.17.1",
  "osVersion": "Windows_NT 10.0.22631",
  "rootPath": "C:\\Users\\JBANNIER\\AppData\\Roaming\\npm\\node_modules\\@salesforce\\cli",
  "shell": "C:\\Program Files\\Git\\usr\\bin\\bash.exe",
  "pluginVersions": [
    "@oclif/plugin-autocomplete 3.0.16 (core)",
    "@oclif/plugin-commands 3.3.1 (core)",
    "@oclif/plugin-help 6.0.21 (core)",
    "@oclif/plugin-not-found 3.1.6 (core)",
    "@oclif/plugin-plugins 5.0.14 (core)",
    "@oclif/plugin-search 1.0.23 (core)",
    "@oclif/plugin-update 4.2.7 (core)",
    "@oclif/plugin-version 2.0.17 (core)",
    "@oclif/plugin-warn-if-update-available 3.0.15 (core)",
    "@oclif/plugin-which 3.1.8 (core)",
    "@salesforce/cli 2.39.6 (core)",
    "apex 3.1.9 (core)",
    "auth 3.6.3 (core)",
    "data 3.3.2 (core)",
    "deploy-retrieve 3.6.6 (core)",
    "info 3.2.3 (core)",
    "limits 3.3.4 (core)",
    "marketplace 1.2.4 (core)",
    "org 4.1.3 (core)",
    "packaging 2.4.2 (user)",
    "schema 3.3.4 (core)",
    "settings 2.2.1 (core)",
    "sobject 1.3.6 (core)",
    "source 3.3.3 (core)",
    "telemetry 3.3.4 (core)",
    "templates 56.2.4 (core)",
    "trust 3.6.6 (core)",
    "user 3.5.2 (core)",
    "sfdmu 4.33.17 (user)",
    "sfdx-essentials 2.12.0 (user)",
    "sfdx-git-delta 5.39.2 (user)",
    "sfdx-hardis 4.33.2 (user)",
    "texei-sfdx-plugin 2.5.0 (user)"
  ]
}

### Actual result
Error (1): _Default: Could not infer a metadata type

[08:56:14.445] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: WebStoreBundle in this organization
[08:56:14.552] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: AssistantVersion in this organization
[08:56:14.553] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: Territory2Rule in this organization
[08:56:14.555] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: OmniIntegrationProcedure in this organization
[08:56:14.558] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: AppointmentSchedulingPolicy in this organization
[08:56:14.659] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: Territory2Type in this organization
[08:56:14.661] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: PermissionSetLicenseDefinition in this organization
[08:56:14.661] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: OmniInteractionAccessConfig in this organization
[08:56:14.662] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Unknown type:ProductSpecificationTypeDefinition
[08:56:14.662] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: AccountingFieldMapping in this organization
[08:56:14.663] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: AccountRelationshipShareRule in this organization
[08:56:14.664] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: AccountingModelConfig in this organization
[08:56:14.664] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: AcctMgrTargetSettings in this organization
[08:56:14.665] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: LiveChatButton in this organization
[08:56:14.665] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: ActionableEventOrchDef in this organization
[08:56:14.723] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: ActionableEventTypeDef in this organization
[08:56:14.792] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: ActionPlanTemplate in this organization
[08:56:14.793] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: ActionableListDefinition in this organization
[08:56:14.795] DEBUG (sf:ConnectionResolver):
    0: "Expected error:"
    1: "Unable to retrieve file for id 1169 (ACSizeTypeCreativeType). This Standard Value Set is either inaccessible or not supported in Metadata API."
[08:56:14.882] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: AdvAccountForecastSet in this organization
[08:56:14.949] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Unknown type:AIAssistantTemplate
[08:56:14.995] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: AIUsecaseDefinition in this organization
[08:56:15.004] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: ActivationPlatform in this organization
[08:56:15.037] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: ActnblListKeyPrfmIndDef in this organization
[08:56:15.139] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: ApplicationRecordTypeConfig in this organization
[08:56:15.141] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: ApplicationSubtypeDefinition in this organization
[08:56:15.211] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: AssessmentQuestionSet in this organization
[08:56:15.667] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: BatchCalcJobDefinition in this organization
[08:56:15.668] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: BenefitAction in this organization
[08:56:15.668] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: ObjectHierarchyRelationship in this organization
[08:56:15.670] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: OmniDataTransform in this organization
[08:56:15.671] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: OcrSampleDocument in this organization
[08:56:15.671] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Unknown type:SvcCatalogItemDefFiltrCrit
[08:56:15.672] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: SvcCatalogFulfillmentFlow in this organization
[08:56:15.672] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: AssistantSkillQuickAction in this organization
[08:56:15.673] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: SvcCatalogFilterCriteria in this organization
[08:56:15.674] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: LiveChatDeployment in this organization
[08:56:15.674] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: TimeSheetTemplate in this organization
[08:56:15.676] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: Territory2 in this organization
[08:56:15.676] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: PricingActionParameters in this organization
[08:56:15.677] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: Territory2Model in this organization
[08:56:15.678] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: RelationshipGraphDefinition in this organization
[08:56:15.678] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: PricingRecipe in this organization
[08:56:15.680] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: AssistantSkillSobjectAction in this organization
[08:56:15.680] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: OmniExtTrackingDef in this organization
[08:56:15.681] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: TransactionSecurityPolicy in this organization
[08:56:15.681] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: PaymentGatewayProvider in this organization
[08:56:15.682] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: SustnUomConversion in this organization
[08:56:15.682] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: ExternalClientApplication in this organization
[08:56:15.685] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Unknown type:EnablementMeasureDefinition
[08:56:15.686] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: FieldSrcTrgtRelationship in this organization
[08:56:15.686] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: CareBenefitVerifySettings in this organization
[08:56:15.687] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: ExternalAIModel in this organization
[08:56:15.687] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: ExternalDataConnector in this organization
[08:56:15.688] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: TimelineObjectDefinition in this organization
[08:56:15.690] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: DataPipeline in this organization
[08:56:15.690] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: CustomDataType in this organization
[08:56:15.690] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: DataSourceBundleDefinition in this organization
[08:56:15.692] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: CallCtrAgentFavTrfrDest in this organization
[08:56:15.694] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Unknown type:ConversationVendorFieldDef
[08:56:15.695] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: CareRequestConfiguration in this organization
[08:56:15.695] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: FieldServiceMobileExtension in this organization
[08:56:15.696] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: UIObjectRelationConfig in this organization
[08:56:15.697] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: RegisteredExternalService in this organization
[08:56:15.697] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: DisclosureDefinitionVersion in this organization
[08:56:15.698] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: EntitlementTemplate in this organization
[08:56:15.699] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: DataPackageKitObject in this organization
[08:56:15.699] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: CareLimitType in this organization
[08:56:15.700] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: CareSystemFieldMapping in this organization
[08:56:15.701] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: SvcCatalogCategory in this organization
[08:56:15.701] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: ESignatureConfig in this organization
[08:56:15.702] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: DataSourceObject in this organization
[08:56:15.725] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: ESignatureEnvelopeConfig in this organization
[08:56:15.726] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: EventSubscription in this version
[08:56:15.726] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: LiveChatAgentConfig in this organization
[08:56:15.727] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: ConversationVendorInfo in this organization
[08:56:15.728] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: DataSource in this organization
[08:56:15.743] DEBUG (sf:ConnectionResolver): INVALID_TYPE: Cannot use: CareProviderSearchConfig in this organization
Error (1): _Default: Could not infer a metadata type

cc @nvuillam

**Comment by mdonnalley** [comment]
`Error (1): _Default: Could not infer a metadata type`

That error message is formatted using: `%s: Could not infer a metadata type` where the `%s` is a file path in your project. Based on that, I'd say that there's a file called `_Default` in your project that shouldn't be there?

If that doesn't help, it'd be helpful to have the entire logs from the failed command and, if possible, a repro that minimally replicates the issue. Thanks!

**Comment by nvuillam** [comment]
@mdonnalley does it help ? :)

Note: the same command is called in all our Monitoring repositories (daily run) so it's really dependent to the org content, not to the way we call the command (it works for 95% of identical jobs based on the same Docker image)

```
Error (1): _Default: Could not infer a metadata type
Try this:
Additional suggestions:
Confirm the file name, extension, and directory names are correct. Validate against the registry at:
<https://github.com/forcedotcom/source-deploy-retrieve/blob/main/src/registry/metadataRegistry.json>
If the type is not listed in the registry, check that it has Metadata API support via the Metadata Coverage Report:
<https://developer.salesforce.com/docs/metadata-coverage>
If the type is available via Metadata API but not in the registry
- Open an issue <https://github.com/forcedotcom/cli/issues>
- Add the type via PR. Instructions: <https://github.com/forcedotcom/source-deploy-retrieve/blob/main/contributing/metadata.md>
```

**Comment by mdonnalley** [comment]
>so it's really dependent to the org content

If that's the case, are you able to isolate the org content that leads to this error? Also, it'd still be helpful to have the full debug output

**Comment by nvuillam** [comment]
@Jbanniere please can you send the full log with --dev-debug after removing the client data ? :)

**Comment by sgupta3SFDC** [comment]
@Jbanniere  Check the metadata type Synonyms in the org as Default gets created for the same and gives error while retrieving the complete manifest. I believe the issue is fixed in the SF CLI version 2.39.6

**Comment by nvuillam** [comment]
@sgupta3SFDC I confirm it seems solved with latest SF Cli version, thanks :)
